Bio/SeqIO/__init__.py

```diff
@@ -438,6 +439,7 @@
                      "fastq-illumina": QualityIO.FastqIlluminaIterator,
                      "qual": QualityIO.QualPhredIterator,
                      "sff": SffIO.SffIterator,
+                     "nib": NibIO.NibIterator,
```

Comment:
This should be a couple of lines higher up (alphabetical), not in the middle of the two SFF entries.